Update namespaces.mdx (#3499)
temporalio/documentation: docs/production-deployment/cloud/account-setup/namespaces.mdx

  Throttling means limiting the rate at which actions are performed to prevent overloading the system.
  A Namespace's default limit is set at 400 APS and automatically adjusts based on recent usage (over the prior 7 days).
  Your APS limit will never fall below this default value.
- Each Namespace has a default service-level agreement (SLA) of 99.9% uptime.

> - However, you can opt-in to using [High Availability](https://docs.temporal.io/cloud/high-availability) features for a 99.99% contractual SLA.

Reviewer: ```suggestion
- You can opt-in to using [High Availability features](https://docs.temporal.io/cloud/high-availability) with a 99.99% contractual SLA.
```
